apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java

```diff
@@ -21,11 +27,26 @@
 public final class ApolloConverterFactory extends Converter.Factory {
   private final Map<Type, ResponseFieldMapper> responseFieldMappers;
   private final Map<ScalarType, CustomTypeAdapter> customTypeAdapters;
+  private final Moshi moshi;
```

Comment:
We use moshi to serialize query request that can have variables (with scalar or input object types).